From daaf1ad338e9a669c67a4701c13573aea0922433 Mon Sep 17 00:00:00 2001
From: chpr <carthango@gmx.net>
Date: Mon, 18 Jan 2021 19:48:40 +0100
Subject: [PATCH 5/5] Configure check for putenv

Additional command putenv shall be checked at configure to avoid
double inclusion on mingw32 build
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index daf857bea..31e5fc0a0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -560,7 +560,7 @@ AC_CHECK_FUNCS_ONCE([lrintf strtof])
 AC_FUNC_FORK
 AC_FUNC_GETGROUPS
 AC_CHECK_FUNCS_ONCE([chmod chown fstat fchown fchmod clock_gettime getaddrinfo getgrgid_r getgrnam_r \
-    getpwnam_r getpwuid_r gettimeofday getuid mlock nanosleep \
+    getpwnam_r getpwuid_r gettimeofday getuid mlock nanosleep putenv \
     pipe posix_fadvise posix_madvise posix_memalign setpgid setsid shm_open \
     sigaction sleep symlink sysconf uname pthread_setaffinity_np pthread_getname_np pthread_setname_np])
 AC_CHECK_FUNCS([mkfifo], [HAVE_MKFIFO=1], [HAVE_MKFIFO=0])
-- 
2.29.2.windows.2

